src/common/PreviewUtils.ts

```diff
@@ -74,11 +75,12 @@ export class PreviewUtils {
     }
 
     public static getAppBundlePath(
+        basePath: string,
         appConfig: IOSAppPreviewConfig | AndroidAppPreviewConfig
     ): string | undefined {
         if (appConfig.get_app_bundle) {
             const module = require(appConfig.get_app_bundle);
-            return module.run();
+            return path.resolve(basePath, module.run());
```

Comment:
This isn't quite right. What I'm saying is that the path *to the module itself* needs to be relative to the path to the config file. Consider the scenario where we're hosting an "internal" config (or configs) and supporting `get_app_bundle` modules in the plugin itself. We're going to end up having a file structure that looks something like this:
```
pluginRootDir/
    apps/
        app_config.json
        ios/
            TestHarnessApp/
                buildAppBundle.js
```
And contents of app_config.json that look like:
```
{
  "apps": {
    "ios": [
      {
        "id": "com.salesforce.mobile-tooling.lwc-test-app",
        "name": "LWC Test App",
        "get_app_bundle": "ios/TestHarnessApp/buildAppBundle.js"
      }
    ]
}
```
We're going to need to be able to build the path to the module, off of the path to the config file, or we won't otherwise have a way to infer its location.

Reply:
Another way to do this would be to create proper Node.js packages to house each these modules (per-app), and install those packages as part of the plugin bootstrap process. I'd need to think through how that would work—but it may be necessary if the requirement as spelled out above proves to be a challenge to implement. There's a fair amount of path manipulation to the approach above, that I haven't fully delved into.